src/components/dialogs/parameters/short-circuit-parameters.jsx

```diff
@@ -65,6 +71,26 @@ export const useGetShortCircuitParameters = () => {
         }
     }, [shortCircuitAvailability, studyUuid, snackError]);
 
+    useEffect(() => {
+        if (
+            studyUuid &&
+            shortCircuitAvailability === OptionalServicesStatus.Up &&
+            studyParamsChanged === ComputingType.SHORT_CIRCUIT
+        ) {
+            getShortCircuitParameters(studyUuid)
+                .then((params) => {
+                    setShortCircuitParams(params);
+                    dispatch(setStudyParamsChanged(''));
+                })
+                .catch((error) => {
+                    snackError({
+                        messageTxt: error.message,
+                        headerId: 'paramsRetrievingError',
+                    });
+                });
```

Comment:
same code duplication...

I understand that we want to fetchparams by notification OR when studyUuid OR shortCircuitAvailability changed. We could write multiple effect to defines those cases, but the consequence should not be duplicated.

Reply:
Done.